apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.desktop.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Alert, Linking, View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { Text } from '@fluentui/react-native';
+import { stackStyle } from '../Common/styles';
+import type { IKeyboardEvent, IHandledKeyboardEvent } from '@office-iss/react-native-win32';
+
+export const PressableUsage: React.FunctionComponent = () => {
+  const _onPress = (): void => {
+    Linking.openURL('https://www.microsoft.com');
+  };
+
+  const _onKeyDown = (ev: IKeyboardEvent) => {
+    if (ev.nativeEvent.key === 'Enter' || ev.nativeEvent.key == 'Space') Linking.openURL('https://www.microsoft.com');
+  };
+
+  const _onPress2 = (): void => {
+    Alert.alert('Alert', 'Success!');
+  };
+
+  const _onKeyDown2 = (ev: IKeyboardEvent) => {
+    if (ev.nativeEvent.key === 'Enter' || ev.nativeEvent.key == ' ') Alert.alert('Alert', 'Success!');
+  };
+
+  const handledNativeKeyboardEvents: IHandledKeyboardEvent[] = [{ key: 'Enter' }, { key: 'Space' }];
```

Comment:
Because macOS uses the prop `validKeysDown` instead of `keyDownEvents`, this won't work on macOS. We have a hook ( `useKey(Up|Down)Props` from `@fluentui-react-native/interactive-hooks`) that returns the right props per platform that can be spread to the final component. I should make a bug to track having it return a `keyDownEvents` array for windows/win32, so that we can use it here.

I also plan to investigate if we can just make the macOS prop align names with the windows prop (`validKeysDown` -> `keyDownEvents`), but that's outside the scope of this PR for sure..